packages/vite/src/node/plugins/css.ts

```diff
@@ -314,7 +315,9 @@ export function cssPostPlugin(config: ResolvedConfig): Plugin {
           if (inlined) {
             return `export default ${JSON.stringify(css)}`
           }
-
+          if (isHTMLProxy) {
+            return css
+          }
```

Comment:
Since CSS transpilation is supported, I think it is better to move this after line 326 and change to `return cssContent` to support sourcemap.

Reply:
So IIUC, this is the last change needed, no?

Reply:
Yes 😃

Reply:
yes, I forgot. 🥹